en/SUMMARY.md

```diff
@@ -16,6 +16,8 @@
 * [Basic Assembly](assembly/README.md)
   * [Mounting the Flight Controller](assembly/mount_and_orient_controller.md)
   * [Vibration Isolation](assembly/vibration_isolation.md)
+  * [V5+ Wiring Quickstart](assembly/quick_start_v5+.md)
```

Comment:
Please prefix all of these in  SUMMARY with "CUAV" - ie "CUAV V5+ Wiring Quickstart", "CUAV V5+" etc.

Reply:
The flight controller is named v5+. Do you think I need to add a manufacturer's trademark?